src/Blocks/components/field/components/field-options-advanced.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { TextControl } from '@wordpress/components';
+import {
+	icons,
+	checkAttr,
+	getAttrKey,
+	IconLabel,
+	ComponentUseToggle
+} from '@eightshift/frontend-libs/scripts';
+import manifest from '../manifest.json';
+
+export const FieldOptionsAdvanced = (attributes) => {
+	const {
+		setAttributes,
+	} = attributes;
+
+	const fieldHelp = checkAttr('fieldHelp', attributes, manifest);
+	const fieldBeforeContent = checkAttr('fieldBeforeContent', attributes, manifest);
+	const fieldAfterContent = checkAttr('fieldAfterContent', attributes, manifest);
+
+	const [showAdvanced, setShowAdvanced] = useState(false);
+
+	return (
+		<>
+			<ComponentUseToggle
```

Comment:
This should be either a regular `ToggleControl` (from Gutenberg) or `IconToggle` (from Frontend libs).